README.adoc

```diff
@@ -246,7 +243,7 @@ InventoryUtil.java
 include::finish/src/main/java/io/openliberty/guides/microprofile/util/InventoryUtil.java[]
 ----
 
-The [hotspot=buildHostJson file=1]`buildHostJson` helper method builds in the `InventoryUtil` class. It
+The [hotspot=buildHostJson file=1]`buildHostJson` helper method builds in the [file=1]`InventoryUtil` class. It
```

Comment:
- [ ] file hotspot still not appearing for `InventoryUtil`
    - need to change to `[file=0]` to `[hotspot file=0]`